grass7/raster/r.learn.ml2/rlearnlib/Makefile

```diff
@@ -13,10 +13,12 @@ PYCFILES := $(patsubst %,$(ETCDIR)/%.pyc,$(MODULES))
 default: $(PYFILES) $(PYCFILES)
 
 $(ETCDIR):
-	$(MKDIR) -p $@
+	$(MKDIR) $@
 
 $(ETCDIR)/%: % | $(ETCDIR)
 	$(INSTALL_DATA) $< $@
 
+
 install:
+	$(MKDIR) $(INST_DIR)/etc/r.learn.ml2
```

Comment:
This one line fix this error (previous changes isn't needed):
`$(MKDIR) $(INST_DIR)/etc/r.learn.ml2/rlearnlib`

Reply:
> I removed all other changes. But I do not need the -p in the Makefile, because $(MKDIR) is in my system mkdir -p and I > am not sure if the -p works in other systems e.g. Windows.

Yes you are right. According `$(MKDIR)` variable definition in the file [Platform.make.in](https://github.com/OSGeo/grass/blob/master/include/Make/Platform.make.in#L31), `mkdir` command has `-p` flag and then no needed extra flag definition in this [Makefile.](https://github.com/OSGeo/grass-addons/pull/364/commits/a1a180b9cb060327d7b412bb60278e2a34bbfaf5#diff-d9941df1d436ecd058e605823ab880424c849dc5531edd81b7b6126ee3fd8ba9R16)